phasespace/fromdecay/genmultidecay.py

```diff
@@ -0,0 +1,290 @@
+import itertools
+from typing import Callable, Union
+
+import tensorflow as tf
+import tensorflow.experimental.numpy as tnp
+from particle import Particle
+
+from phasespace import GenParticle
+
+from .mass_functions import _DEFAULT_CONVERTER
+
+_MASS_WIDTH_TOLERANCE = 0.01
+_DEFAULT_MASS_FUNC = "relbw"
+
+
+class GenMultiDecay:
+    def __init__(self, gen_particles: list[tuple[float, GenParticle]]):
+        """A container that works like GenParticle that can handle multiple decays. Can be created from.
+
+        Args:
+        gen_particles: All the GenParticles and their corresponding probabilities.

```

Comment:
I think you need to indent after `Args:`. Also,  isn't an empy line needed? It may depend on the style of docstring (NumPy or Google).